### PR TITLE
Regular expression fixes

### PR DESCRIPTION
**Pull request by yavyx**
- Fixed morphology regex pattern
- Simplified the other patterns
- Removed unused code

### CODE REVIEW

File: chord_metadata_service/mohpackets/permissible_values.py [hunk 1]
```diff
@@ -1086,47 +1086,27 @@
...
 # A date, or partial date (e.g. just year or year + month) as used in
 # human communication. The format is YYYY, YYYY-MM, or YYYY-MM-DD,
 # e.g. 2018, 1973-06, or 1905-08-23. There SHALL be no time zone.
-DATE_REGEX_PATTERNS = r"^([0-9]([0-9]([0-9][1-9]|[1-9]0)|[1-9]00)|[1-9]000)(-(0[1-9]|1[0-2])(-(0[1-9]|[1-2][0-9]|3[0-1]))?)?"  # noqa: E501
+DATE_REGEX = r"^([0-9]([0-9]([0-9][1-9]|[1-9]0)|[1-9]00)|[1-9]000)(-(0[1-9]|1[0-2])(-(0[1-9]|[1-2][0-9]|3[0-1]))?)?"  # noqa: E501
 
 # ICD-O-3 morphology codes
 # Examples: 8260/3, 9691/36
-MORPHOLOGY_REGEX_PATTERNS = r"^[8,9]{1}[0-9]{3}/[0,1,2,3,6,9]{1}[1-9]{0,1}$"
+MORPHOLOGY_REGEX = r"^\d{4}[/]\d{1}$"
```
Comment (SonQBChau) [inline chord_metadata_service/mohpackets/permissible_values.py]: Is the old regex wrong ? Do ICD-O-3 morphology codes accept any number?

Comment (yavyx) [reply chord_metadata_service/mohpackets/permissible_values.py]: Reverted back to the original pattern, all of the regexes worked as expected in the end :neutral_face:

Comment (SonQBChau) [reply chord_metadata_service/mohpackets/permissible_values.py]: is this PR necessary?

Comment (yavyx) [reply chord_metadata_service/mohpackets/permissible_values.py]: It cleans up some unused code

Comment (SonQBChau) [reply chord_metadata_service/mohpackets/permissible_values.py]: I will put this PR after katsu date interval then, since it just cleans up

**Comment by SonQBChau** [comment]
Can you fix the conflict so we can merge into develop?

**Pull request merged**